pkg/extension/workspace/toolchain/builtin/default_config.go

```diff
@@ -69,6 +72,8 @@ func defaultOutputPathFor(language extensionconfig.Language) string {
 	case extensionconfig.LanguageRust:
 		// keep *.wasm file inside Cargo build dir (to be cleaned up automatically)
 		return "target/getenvoy/extension.wasm"
+	case extensionconfig.LanguageTinyGo:
+		return "extension.wasm"
```

Comment:
Can we move generated extension file into `build/extension.wasm` ?

Reply:
fixed in https://github.com/tetratelabs/getenvoy/pull/104/commits/b9024b03272fdc0154bc100bbf84b6978afb8b1c